src/components/ModalPayment/SquareModal.tsx

```diff
@@ -352,10 +353,6 @@ const CheckboxContainer = styled.label`
   :hover {
     text-decoration: underline;
   }
-
-  > span {
-    padding: 9px 9px 9px 0px;
-  }
```

Comment:
Removing this fixes the misshapen checkbox effect on prod: 
![image](https://user-images.githubusercontent.com/11382902/92957791-23d8a280-f437-11ea-8fc0-3efae9a340b9.png)